instrumentation/active_record/test/instrumentation/active_record/patches/querying_test.rb

```diff
@@ -17,10 +17,16 @@
 
   describe 'find_by_sql' do
     it 'traces' do
+      Account.create!
```

Comment:
Would this updated test fail on 7.0 without the change to `patches/querying.rb`? If not, what do you think about adding a test to cover the scenario that drove this fix?

Reply:
If I comment the changes to the patch, the test fails in 7.0, 7.1 and 7.2 with a:

```
  1) Failure:
OpenTelemetry::Instrumentation::ActiveRecord::Patches::Querying::find_by_sql#test_0001_traces [test/instrumentation/active_record/patches/querying_test.rb:29]:
Expected nil to not be nil.
```

Because the `Account.first` doesn't produce a span. I'd say the test is redgreening the problem and works across the supported versions. Would you prefer there's one test with the explicit `User.find_by_sql` and one that tests the other types of reads (the `Account.first.users`)?